plugins/woocommerce/tests/e2e-pw/tests/merchant/create-simple-product.spec.js

```diff
@@ -111,8 +114,44 @@ test.describe.serial( 'Add New Simple Product Page', () => {
 		await page.goto( 'wp-admin/post-new.php?post_type=product', {
 			waitUntil: 'networkidle',
 		} );
-		await page.locator( '#title' ).fill( nonVirtualProductName );
+		await page.getByLabel( 'Product name' ).fill( nonVirtualProductName );
```

Comment:
@Stojdza , @lanej0 I was struggling to get how I could get inside the iframe's text area to add a product description. I would like to ask you if it is okay if I pick these lines to reuse them for https://github.com/woocommerce/woocommerce-quality/issues/607 ?